kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -384,4 +365,32 @@ public Subscription create(Subscription subscription) {
     }
 
 
+    @Override
+    public final void createSubscriptionPacks() {
+        if (subscriptionPackDataService.byName("childPack") == null) {
+            createSubscriptionPack("childPack", SubscriptionPackType.CHILD, CHILD_PACK_WEEKS, 1);
+        }
+        if (subscriptionPackDataService.byName("pregnancyPack") == null) {
+            createSubscriptionPack("pregnancyPack", SubscriptionPackType.PREGNANCY, PREGNANCY_PACK_WEEKS, 2);
+        }
+    }
+
+
+    private void createSubscriptionPack(String name, SubscriptionPackType type, int weeks,
```

Comment:
I thought we were not creating these and it was going to be an OPS task.
Did I miss the memo?
![Did I miss the memo?](http://www.troll.me/images/office-space-boss/yeeaahhhhh-did-you-see-the-memo-about-this.jpg)